irohad/ordering/impl/on_demand_ordering_service_impl.cpp

```diff
@@ -116,8 +116,9 @@ getTransactions(size_t requested_tx_amount,
   std::vector<std::shared_ptr<shared_model::interface::Transaction>> collection;
 
   auto it = batch_collection.begin();
-  for (;
-       it != batch_collection.end() and collection.size() < requested_tx_amount;
+  for (; it != batch_collection.end()
+       and collection.size() + boost::size((*it)->transactions())
```

Comment:
I suggest using brackets for plus:
```suggestion
       and (collection.size() + boost::size((*it)->transactions()))
```